Ginger/GingerCoreNET/Database/NoSqlBase/GingerCassandra.cs

```diff
@@ -37,32 +42,72 @@ public class GingerCassandra : NoSqlBase
         dynamic myclass = null;
         string mUDTName = null;
 
-
-
         public override bool Connect()
         {
             try
             {
-                string queryTimeoutString = "querytimeout=";
+                const string queryTimeoutString = "querytimeout=";
+                const string sslString = "ssl=";
                 int queryTimeout = 20000;//default timeout (20 seconds).
-                if (Db.DatabaseOperations.TNSCalculated.ToLower().Contains(queryTimeoutString.ToLower()))
+                string[] queryArray = Db.DatabaseOperations.TNSCalculated.Split(';');
```

Comment:
Since you are not comparing the case in the switch case below, make sure you get the string .ToLower() on line #52